EXO-59649 : Redaction of the contents of the page Simple integration (#2)
exoplatform/exoplatform.github.io: docs/guide/developer-guide/simple-integration/README.md

# Simple integration 

> eXo platform is offering many capabilities for developers to integrate their developments and customize its default functionality set.

Reviewer: ```suggestion
eXo Platform offers many capabilities for developers to integrate their developments and customize its default functionality set.
```

> To start developing on top of eXo platform, we need to start by creating a new empty project that will represent an eXo extension.

Reviewer: ```suggestion
To start developing on top of eXo Platform, you need to start by creating a new empty project for an eXo extension.
```

## eXo extension

### Why an eXo extension

> An eXo extension covers almost all kind of customization needed by developers :

Reviewer: ```suggestion
eXo extensions let you customize eXo Platform  for :
```
